code/controllers/subsystem/overlays.dm

```diff
@@ -71,6 +72,9 @@ SUBSYSTEM_DEF(overlays)
 		count = 0
 
 /proc/iconstate2appearance(icon, iconstate)
+	if (istype(icon, /icon))
+		var/icon/I = icon
```

Comment:
Single-letter variable names are not allowed anymore as per https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names.

`icon_icon` here would be a bit silly though, perhaps this can just be `icon_casted`.

Reply:
I don't plan on obeying this so you can maintainer edit this.